.github/workflows/bump-version-release.yaml

```diff
@@ -57,34 +52,35 @@ jobs:
         uses: bakdata/ci-templates/actions/checkout@1.32.0
         with:
           ref: ${{ github.event.repository.default_branch }}
-          persist-credentials: false # required for pushing to protected branch later
+          persist-credentials: true # required for pushing to protected branch later
```

Comment:
Why did this change? I think this would lead to the CI being logged in with default credentials and not with the bot user